lld/test/COFF/gnu-weak.test

```diff
@@ -1,4 +1,9 @@
 RUN: lld-link -lldmingw %S/Inputs/gnu-weak.o %S/Inputs/gnu-weak2.o -out:%t.exe
+RUN: lld-link -lld-allow-duplicate-weak %S/Inputs/gnu-weak.o %S/Inputs/gnu-weak2.o -out:%t.exe
+RUN: not lld-link %S/Inputs/gnu-weak.o %S/Inputs/gnu-weak2.o -out:%t.exe 2>&1 | FileCheck %s --check-prefix=DEFAULT-ERROR
+
+DEFAULT-ERROR: error: duplicate symbol: weakfunc
```

Comment:
If this is the last line, consider adding `DEFAULT-ERROR-NOT: {{.}}` as a change detector when the diagnostic improves.

Reply:
There are more lines following this diagnostic (`defined at /path/to/object/file`), so I'll leave this as is.